src/foam/nanos/ticket/TicketDAOCommand.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.ticket',
+  name: 'TicketDAOCommand',
```

Comment:
rename to 
TicketCloseCmd or TicketCloseCommand
it does not need the constant
just the ticket id (reference)
The cmd_ of TicketCmdDAO will test for a instanceof TicketCloseCmd.

Reply:
My intention was for this class to be generic so that we can reuse this class if we want to add a new command for ticket dao. What do you think?

Reply:
no.